ydb/core/tx/datashard/change_sender_common_ops.cpp

```diff
@@ -584,16 +579,11 @@ void TBaseChangeSender::RenderHtmlPage(TEvChangeExchange::ESenderType type, NMon
                             TABLER() {
                                 TABLED() { html << ++i; }
                                 TABLED() { html << order; }
-                                TABLED() { html << record.GetGroup(); }
-                                TABLED() { html << record.GetStep(); }
-                                TABLED() { html << record.GetTxId(); }
-                                TABLED() { html << record.GetLockId(); }
-                                TABLED() { html << record.GetLockOffset(); }
-                                TABLED() { PathLink(html, record.GetPathId()); }
```

Comment:
Is it necessary to remove this debug info?

Reply:
Yes, for now. Maybe i'll return it later.